src/package/external/external.csproj

```diff
@@ -50,7 +50,7 @@
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Internal.TestPlatform.Extensions">
-      <Version>$(TestPlatformExternalsVersion)-patched7</Version>
```

Comment:
Please upgrade also other packages that we get from mstest to TestPlatformExternalsVersion.

In this file we have:
Microsoft.Internal.Intellitrace
Microsoft.Internal.Dia

In other places we reference:
Microsoft.Internal.CodeCoverage
Microsoft.Internal.Dia.Interop

We should upgrade all of them together. Otherwise we can have some issues

Reply:
Is it not? This line is just removing the suffix that was added to be able to use a different version. Are the dependencies that you mentioned using the variable that is used here?